packages/core/documentation/src/components/DocumentationPageEmptyState.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <KEmptyState
+    class="documentation-page-empty-state"
+    data-testid="documentation-page-empty-state"
+    icon="stateGruceo"
+    icon-size="96"
+  >
+    <template #title>
+      {{ i18n.t('documentation.show.empty_state.title') }}
+    </template>
+    <template #message>
+      {{ i18n.t('documentation.show.empty_state.message') }}
+    </template>
+    <template #cta>
+      <PermissionsWrapper
+        :auth-function="() => canEdit()"
+      >
+        <KButton
+          appearance="primary"
+          data-testid="add-new-page-button"
+          icon="plus"
```

Comment:
Can you just slot in an icon from `@kong/icons`? This prop will be going away next week

Reply:
fixed